includes/Core/Modules/Modules.php

```diff
@@ -1550,7 +1551,7 @@ function( $module ) {
 	 * @param array $sharing_settings The dashboard_sharing settings option fetched from the database.
 	 * @return array Dashboard sharing settings option with default settings inserted for shared ownership modules.
 	 */
-	protected function filter_shared_ownership_module_settings( $sharing_settings ) {
+	protected function populate_default_shared_ownership_module_settings( $sharing_settings ) {
```

Comment:
This JSDoc block above this should be updated, the description is now a little out of sync. It would also benefit from a `@since n.e.x.t` entry to describe the name change.

Reply:
Updated, thank you @techanvil!